go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/gopasspw/gopass
 
 go 1.23.2
 
+replace github.com/pquerna/otp => github.com/nikicat/otp v1.4.0-ojo
```

Comment:
This would be too much of a supply chain attack risk, and also replace statement in a go.mod aren't meant to be shipped in a library, and some bits of gopass are acting as a library, so mot great.

Any chance you could upstream that to pquerna otp code or provide the derivation in our code directly?

We're trying to generally reduce the number of dependencies, not add new ones.

Reply:
I'll try to merge this change upstream, that's why I've created a draft PR. I think conversion from a standard digital code format is possible, but is a bit hacky, as it needs to requires at least 8 digits code to be requested and then reencoded to Steam's base26 format.

Reply:
Yeah, sorry I fat fingered the undraft button when trying to authorize the Github Action